### DISCUSSION TITLE
Remove sensitive information when replying/sending to reverse alias

### DISCUSSION THREAD

**Discussion by johannrichard** [simple-login/app#1103] [Feature Requests]
While starting to use SimpleLogin, I came around use cases where the privacy of my destination mailbox can easily be thwarted by the user themselves. Namely, there are some cases where a user might reply to a reverse alias with a message that contains the original destination e-mail address: 

— an automatic out-of-office message, 
— an e-mail signature, or
— an S/MIME certificate attached to an outgoing message.

It would be desirable if any occurrence of the destination e-mail mailbox address would either be replaced by the alias or otherwise anonymized. Similar, if for whatever reason I would be sending out a message with an S/MIME (or PGP) *signature* attached, it would be desirable to remove these attachments or refuse delivery altogether. 

As far as I can tell, such a feature request has not yet been made.

**Comment by strawrhu** [comment]
This is a good point. It is all too easy to _accidentally_  unmask yourself when sending aliased email.

**Comment by eopo** [comment]
Does exist as experimental feature.

**Reply by Pencilcaseman** [reply]
How can I enable this?
